### ISSUE TITLE
NVDA screenreader loses focus when opening a file or folder

### ISSUE THREAD

**Issue by gFrincu** [microsoft/vscode#204623]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.86.0
- OS Version: Windows 10 22H2 (AMD64) build 19045.3930 + NVDA 2023.3.3

Steps to Reproduce:

1. Open vs code using NVDA
2. Open a folder and try to open the menu bar, the explorer, the palete...
3. Do alt tab and come to the window again. Observe that you can't do anything,
4. Restart NVDA and see that it works.

This happens with files and folders but doesn't happen always. I would say that it's first time that you open a folder when using vscode and then it stops, but not sure. Also sometimes the focus gets lost while coding, but at least that happens rarely than this folder thing. I am not sure if it's a NVDA or vscode issue

**Comment by meganrogge** [comment]
thanks for the issue. since it seems like an intermittent issue, and I cannot reproduce, I'd suggest creating an issue for NVDA, as that is likely the culprit.
